spec/jsSpecs/ares-device.spec.js

```diff
@@ -120,6 +121,15 @@ describe(aresCmd, function() {
 });
 
 describe(aresCmd + ' --resource-monitor(-r)', function() {
+    beforeAll(function(done) {
```

Comment:
I think beforeAll is enough.
how about delete afterAll?
and...I think beforeall is mean first of all tc, right?

Reply:
Done.